plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -203,6 +203,27 @@ private function get_feature_definitions() {
 						return $string;
 					},
 				),
+				'experimental_blocks'  => array(
+					'name'            => __( 'Experimental blocks', 'woocommerce' ),
+					'description'     => __( 'Try blocks that are in an experimental state', 'woocommerce' ),
+					'is_experimental' => true,
+					'disable_ui'      => false,
+					'is_legacy'       => true,
```

Comment:
This is_legacy option is important to be set to true, otherwise redundant warnings are displayed in settings. I think FeaturesController maybe needs a bit of a revamp for current use cases 😄